mcs/class/System/Mono.Btls/MonoBtlsObject.cs

```diff
@@ -102,6 +102,20 @@ protected void CheckError (int ret, [CallerMemberName] string callerName = null)
 			CheckError (ret == 1, callerName);
 		}
 
+		protected internal void CheckLastError ([CallerMemberName] string callerName = null)
```

Comment:
Does it have to here when it's called from one place only?
Could also be make static